packages/xo-web/src/common/xo/copy-vms-modal/index.js

```diff
@@ -13,25 +14,41 @@ import { connectStore } from 'utils'
 
 import SelectCompression from '../../select-compression'
 import ZstdChecker from '../../zstd-checker'
-import { createGetObjectsOfType } from '../../selectors'
+import {
+  createGetObject,
+  createGetObjectsOfType,
+  createSelector,
+} from '../../selectors'
 
 @connectStore(
   () => {
-    const getVms = createGetObjectsOfType('VM').pick((_, props) => props.vms)
+    const getVms = createGetObjectsOfType((_, props) => props.type).pick(
+      (_, props) => props.vms
+    )
+    const getIsZstdSupported = createSelector(
+      createGetObject(
+        createSelector(getVms, vms =>
+          vms.length === 1 ? Object.values(vms)[0].$container : undefined
+        )
+      ),
+      container => container === undefined || container.zstdSupported
+    )
```

Comment:
Why do you check this now? Why don't you keep the previous behaviour?

Reply:
It's the same behavior when copying a VM.